### ISSUE TITLE
Tasks/Lists: not sortable via keyboard

### ISSUE THREAD

**Issue by fraulueneburg** [fraulueneburg/sortof#3]
Tasks and lists should be draggable/droppable via keyboard.

**Problem here:**
Adding @dnd-kits KeyboardSensor alone won’t work since this makes all items **inside** a draggable element inaccessible to keyboard users (i.e. toggling the checkmark via space bar is no longer possible no matter is a task is currently dragged or not) 👉 solve this
